scheduler-jobs/deploy_data_specific_image

```diff
@@ -88,10 +88,11 @@ for ((i=0;i<$CONFIG_LIST_LENGTH;i++)); do
 
     # Synchronize source_dir with dest_dir
     # --delete : delete extraneous files from destination dirs
-    rsync -rtovcs --delete $FULL_TMP_DIR $FULL_DEST_DIR/
+    rsync --recursive --delete --times --checksum --protect-args $FULL_TMP_DIR/ $FULL_DEST_DIR/
```

Comment:
Just curious, the `-ov` in previous `-rtovcs` are not needed anymore?

Reply:
The `o` stands to preserve ownership a files, but this would require to be run as root, so I removed it because it does not apply in our case.
The `v` was for verbose while I was testing, and I dont know if it would be usefull in our case, it shows all the files copied, but them I dont know if it would overload the logs with information.
Should we keep the `-v` (verbose)?

Reply:
I think `-v` will only display files overridden so should not spam the logs that much.